api/src/main/java/com/aldoborrero/tinder/api/model/TinderEndpoint.java

```diff
@@ -0,0 +1,36 @@
+package com.aldoborrero.tinder.api.model;
+
+import retrofit.Endpoint;
+
+public class TinderEndpoint implements Endpoint {
```

Comment:
I prefer to have fixed constants, TinderEndpoint as Production for example where url is fixed. In here you have to create and specify an url and a name (we can have this option to the user in order to allow using custom endpoints, for development or so). But at least we have ProductionTinderEndpoint (the default one in the builder).